### PR TITLE
Adding Rollbar  Resource

### PR DESCRIPTION
**Pull request by rajdnp**
*Issue #, if available:*

*Description of changes:*
 Adding Rollbar -Projects-Project  Resource changes. 

Tested changes in local in alpha ,beta and Prod. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/rollbar/cicd.yml [hunk 1]
```diff
@@ -0,0 +1,743 @@
...
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: codebuild.amazonaws.com
+        Version: '2012-10-17'
+      ManagedPolicyArns:
+        - Fn::ImportValue: !Sub "cep-${Env}-common-build-project-policy"
+
+  CommonBuildProjectPolicy:
```
Comment (ericzbeard) [inline release/rollbar/cicd.yml:98]: This is not necessary. We moved it to `common.yml`.

File: release/rollbar/cicd.yml [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - sts:AssumeRole
+                Resource:
+                  - !Sub "arn:aws:iam::${AWS::AccountId}:role/AWSCloudFormationStackSetExecutionRole"
+
+  StackSetExecutionRole:
```
Comment (ericzbeard) [inline release/rollbar/cicd.yml:598]: We don't need anything for stack sets, that was left over from when we were using stack sets to publish

File: release/rollbar/release.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#
+#   $1 release number, for example 0.1.0
+
+set -eou pipefail
+
+RELEASE_NUMBER=$1
+FILE_NAME=release-${RELEASE_NUMBER}.zip
+ACCOUNT_ID=$(aws sts get-caller-identity|jq -r .Account)
+
+curl --output-dir /tmp -LO https://github.com/rajdnp/cloudformation-rollbar-resource-providers/archive/refs/tags/$FILE_NAME
```
Comment (ericzbeard) [inline release/rollbar/release.sh:21]: This should be the actual repo

File: release/rollbar/cicd.yml [hunk 4]
```diff
@@ -0,0 +1,743 @@
...
+                Provider: S3
+                Version: 1
+              Configuration:
+                S3Bucket: !Ref SourceBucket
+                S3ObjectKey: "source.zip"
+                PollForSourceChanges: true
+              OutputArtifacts:
+                - Name: extensions-source
+        - Name: Build
+          Actions:
```
Comment (ericzbeard) [inline release/rollbar/cicd.yml:381]: There should be more projects here, one for each resource.

**Review by ericzbeard — CHANGES_REQUESTED** [review]

File: release/rollbar/cicd.yml (hunk 4 above)
Comment (rajdnp) [reply release/rollbar/cicd.yml:381]: Other Resources  will  be added as part of future PRs when those are added later this week.

**Review by ericzbeard — APPROVED** [review]

**Review by ericzbeard — APPROVED** [review]

**Review by ericzbeard — APPROVED** [review]

**Pull request merged**